src/env/dev/terraform.tfvars

```diff
@@ -348,6 +348,9 @@ bpd_hostname       = "api.dev.cstar.pagopa.it"
 cobadge_hostname   = "portal.test.pagopa.gov.it/pmmockserviceapi"
 fesp_hostname      = "portal.test.pagopa.gov.it"
 satispay_hostname  = "portal.test.pagopa.gov.it/pmmockserviceapi"
+cstar_ip_1         = "20.105.180.187"
```

Comment:
are there ip outbound of lb of aks-cstar ?

Reply:
outbound, I update variable name, thanks